1906-11-12.xml

```diff
@@ -56,1798 +60,4978 @@
+                            Opera House. Lift. Electric Light Throughout. Perfect Sanitary
+                            Arrangements. Magnificent Ball, Reception, Reading, and Music Rooms. Bar
+                            and Smoking Room.</p>
+                        <p> HENRI CHAMOULLEAU, Proprietor.</p>
+                        <p>45</p>
+                        <p>FINE TERRACE ON THE AVENUE. - SPLENDID GARDEN. - OMNIBUS MEET ALL TRAINS
+                            AND STEAMERS. <measure type="indexNo">28-26</measure></p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-aan03">
+                        <head>Hamburg &amp; Anglo-American Nile Co.</head>
+                        <p>Weekly departure during Winter Season by the<lb/> Luxurious First Class
+                            Tourist Steamers VICTORIA, PURITAN &amp; MAYFLOWER.<lb/> Regular weekly
+                            Departures to the SECOND CATARACT by the S.S. INDIANA.<lb/> THROUGH
+                            BOOKINGS TO KHARTOUM, GONDOKORO AND THE WHITE NILE.<lb/> Steamers and
+                            Dahabeahs for private charter. Steam Tugs and Steam Launches for
+                            hire.<lb/> FREIGHT SERVICE BY STEAM BARGES BETWEEN CAIRO AND
+                            ALEXANDRIA.<lb/> Working in conjunction and under special arrangement
+                            with the<lb/> “Upper Egypt Hotels Company."</p>
+                        <p>For details and illustrated programmes apply to "THE HAMBURG and
+                            ANGLO-AMERICAN<lb/> NILE COMPANY."</p>
+                        <p> OFFICES IN CAIRO: Sharia Boulac, "Grand Continental Hotel Buildings.”
+                                <measure type="indexNo">31-3-07</measure></p>
+                        <p>Hamburg-Amerika Linie.</p>
+                        <p>Return to Egypt<lb/> by the new S.S. Oceana of 9000 Tons, 350 berths
+                            including 90 single cabins, greatest comfort, best cuisine.</p>
+                        <p>FARES: Genoa to Alexandria from £14. Naples to Alexandria from £10.</p>
+                        <table>
+                            <row>
+                                <cell>Leave Genoa</cell>
+                                <cell><date when="1906-11-23">November 23rd</date> p.m.</cell>
+                                <cell>arr. Alexandria</cell>
+                                <cell><date when="1906-10-27">October 27</date> a.m.</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-06">November 6th</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-10">November 10</date> ,,</cell>
+                            </row>
+                            <row>
+                             
```

Comment:
this and following items are part of the item above (on Capt Waldberg), and shouldn't be split into separate divs.